<launch>
    <node name="lane_detection_node" pkg="darkscnn_caffe" type="darkscnn_caffe_node" output="screen">
        <param name="model_path" value="$(find darkscnn_caffe)/data/deploy.prototxt" />
        <param name="weight_path" value="$(find darkscnn_caffe)/data/deploy.caffemodel" />
        <param name="confidence_threshold" value="0.95"/>
        <param name="alpha" value="0.75"/>
    </node>


    <!-- image tranport, commpressed in, raw out -->

    <node name="republish" type="republish" pkg="image_transport" output="screen" args="compressed in:=/pylon_camera_node_center/image_rect raw out:=/input/image" />

</launch>
